internal/rbd/nodeserver.go

```diff
@@ -1396,9 +1400,42 @@ func getDeviceSize(ctx context.Context, devicePath string) (uint64, error) {
 	return size, nil
 }
 
-func (ns *NodeServer) SetReadAffinityMapOptions(crushLocationMap map[string]string) {
+// getReadAffinityMapOptionFromConfigMap retrieves read affinity map options for cluster `clusterID`.
+func (ns *NodeServer) getReadAffinityMapOptionFromConfigMap(clusterID string) string {
+	crushLocationLabels, err := util.GetReadAffinityOptions(util.CsiConfigFile, clusterID)
+	if err != nil {
+		log.FatalLogMsg(err.Error())
```

Comment:
it should not be Fatal you need to return the error message from the function for all cases